sync/block_listener.go

```diff
@@ -65,6 +65,10 @@ func (bl *BlockListener) ListenToGossipBlocks() {
 			bl.Log.Info("listening  stopped")
 			return
 		case data := <-bl.receivedGossipBlocks:
+			if !bl.IsLatest() {
```

Comment:
we check this later in the method, we also decided to keep these blocks in the database (no propagating)
so we don't have to fetch them again

Reply:
what if they're bad blocks? our db could be flooded

Reply:
its an optimisation traffic 
there are ways of handling the problem you suggested like limiting the size